src/util.test.ts

```diff
@@ -33,27 +34,41 @@ describe("Util", (): void => {
       ): any => {
         callback(error, { stdout, stderr });
       }));
-      const output = await util.execBashCommand(command);
+
+      const output = await util.execBashCommand(command, platform);
 
       expect(core.info).nthCalledWith<[string]>(1, command);
+      const shell =
+        platform === "win32"
+          ? "C:\\Program Files\\Git\\bin\\bash.exe"
+          : "/usr/bin/bash";
       expect(child_process.exec).lastCalledWith(
         command,
-        { shell: "/usr/bin/bash" },
+        { shell },
         expect.anything()
       );
       return output;
     };
 
+    const platform = () => constantFrom<NodeJS.Platform>("linux", "win32");
+
     testProp(
       "ferries command output to GitHub Actions on success",
-      [string(), string(), string()],
+      [string(), string(), string(), platform()],
```

Comment:
Multiple instances: we should provide at least two explicit test cases like we do [here](https://github.com/ScribeMD/docker-cache/blob/3c74f60e0b0e1ec4c30fed29f7627e58e714bd0d/src/docker.test.ts#L271-L278) in order to ensure that we don't fail test coverage if `constantFrom` happens to pull the same element every time.